docs/assets/css/tabs.less

```diff
@@ -1,166 +1,103 @@
-/* ==========================================================================
-   tabs
-   ========================================================================== */
-// Based on gov.uk Tabs: https://design-system.service.gov.uk/components/tabs/
-// breakpoints:
-// 40.0525em
-// 40.0625em; 641px
-// if we change them in this CSS we also have to fork and release our own JS with updated breakpoints
-
-
-@media (min-width: 40.0625em) {
-    .govuk-tabs {
-        margin: 0 -1rem;
-    }
-}
-
-.govuk-tabs__list {
-    border-top: 1px solid @gray;
+.m-tabs {
+    width: 20em;
+    font-family: "lucida grande", sans-serif;
+  }
+  
+  [role="m-tabs__list"] {
```

Comment:
`role` will be an [aria role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques) from a predefined set, so we can't arbitrarily set it to a class name.

In the [W3 example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html) they have `role="tab"` and doing the matching on that because they don't have class names defined. You do have class names so can do the matching on that, so `.m-tabs_list {` (note I've converted it into an element syntax)